### ISSUE TITLE
add minibatches to all models

### ISSUE THREAD

**Issue by epicgamer17** [epicgamer17/rl-research#60]
like ppo, training on multiple minibatches of data per training step (would this just be similar to just increasing batch size and learning rate?). Actually I think it would be slightly different as after each minibatch the model has learned so the results would be slightly different so loss might be less (ei it would not be the same as doubling the learning rate and thus original loss)
